apps/wallet-dashboard/components/dialogs/staking/views/DetailsView.tsx

```diff
@@ -77,7 +78,11 @@ export function DetailsView({
     const [iotaEarnedFormatted, iotaEarnedSymbol] = useFormatCoin({ balance: iotaEarned });
     const [totalStakeFormatted, totalStakeSymbol] = useFormatCoin({ balance: totalStake });
 
-    const validatorName = validatorSummary?.name || '--';
+    const { data: inactiveValidatorSummary } = useGetInactiveValidator(validatorAddress);
+    const validatorName =
+        inactiveValidatorSummary?.name || validatorSummary?.name || validatorAddress;
+    const validatorImageUrl =
+        inactiveValidatorSummary?.imageUrl || validatorSummary?.imageUrl || null;
```

Comment:
why use the inactiveValidatorSummary first instead of using the normal one and fallback to inactive?